addons/website_sale_loyalty/static/src/scss/website_sale_loyalty.scss

```diff
@@ -0,0 +1,23 @@
+table#cart_products, table#sales_order_table {
+    .badge {
+        background-color: #FFCC00;
+        border: 1px solid #B8860B;
+    }
+
+    .points {
+        color: #267372;
```

Comment:
The color values are hardcoded here, maybe they should be defined as css variables or use existing ones from the website ?